media/redesign/js/wiki.js

```diff
@@ -578,6 +578,21 @@
         }
     });
 
+    /*
+      Make Compare selected revisions button sticky when scrolling history page
+    */
+    (function() {
+        var $revisionButtonOffset = $('.revision-list-controls .link-btn').offset().top;
+        $(window).scroll( function() {
+            var $compareButton = $('.revision-list-controls .link-btn');
+            var $scroll = $(window).scrollTop();
+            if ($scroll >= $revisionButtonOffset) {
+                $compareButton.addClass('fixed');
+            } else {
+                $compareButton.removeClass('fixed');
+            }
```

Comment:
That makes sense @robhudson, I'll try to opt for the more compact toggle/toggleClass methods where it makes sense in the future.